lib/resty/acme/autossl.lua

```diff
@@ -145,7 +147,7 @@ local function get_certkey_parsed(domain, typ)
   elseif err_ret and data_staled then
     log(ngx_WARN, err_ret, ", serving staled cert for ", domain)
     return data_staled, nil
-  else
+  elseif not skip_null_cache then
```

Comment:
this *should* be already handled? if no, let's actively purge the null cache after a successful cert creation.

Reply:
Mmh it's not hanlded already I think, because due to the failed first get cert attempt there's `null` in cache. So on the next get cert attempt after cert creation the code gets the cached `null` cert. And we get the debug message `negative cached domain cert`.

But I agree, purging the cache is better, I've changed that.